etc/dev/nginx/nginx.conf

```diff
@@ -8,11 +8,13 @@ server {
         try_files $uri /index.php$is_args$args;
     }
 
-    location ~ ^/index\.php(/|$) {
+    location ~ \.php$ {
```

Comment:
For some security reasons I prefer to keep just index here. Do you agree?

Reply:
I don't really know nginx config but keeping index.php makes PHPStorm fail at validating xdebug because it can't access a script file put in /public.
If you have any workaround for this, I'm open to suggestions :)

Reply:
I this can be done for the dev env to allow phpstorm validation